lib/src/helpers.dart

```diff
@@ -180,3 +206,83 @@ num convertArea(num area,
 
   return (area / startFactor) * finalFactor;
 }
+
+
+/// Converts coordinates from one system to another.
+///
+/// Valid systems: [CoordinateSystem.wgs84], [CoordinateSystem.mercator]
+/// Returns: [List] of coordinates in the target system
+List<double> convertCoordinates(
+  List<num> coord, 
+  CoordinateSystem fromSystem, 
+  CoordinateSystem toSystem
+) {
+  if (fromSystem == toSystem) {
+    return coord.map((e) => e.toDouble()).toList();
+  }
+  
+  if (fromSystem == CoordinateSystem.wgs84 && toSystem == CoordinateSystem.mercator) {
+    return toMercator(coord);
+  } else if (fromSystem == CoordinateSystem.mercator && toSystem == CoordinateSystem.wgs84) {
+    return toWGS84(coord);
+  } else {
+    throw Exception("Unsupported coordinate system conversion: $fromSystem to $toSystem");
+  }
+}
+
+/// Converts a WGS84 coordinate to Web Mercator.
+///
+/// Valid inputs: [List] of [longitude, latitude]
+/// Returns: [List] of [x, y] coordinates in meters
+List<double> toMercator(List<num> coord) {
+  if (coord.length < 2) {
+    throw Exception("coordinates must contain at least 2 values");
+  }
+  
+  // Use the earth radius constant for consistency
+  
+  // Clamp latitude to avoid infinite values near the poles
+  final longitude = coord[0].toDouble();
+  final latitude = max(min(coord[1].toDouble(), 89.99), -89.99);
+  
+  // Convert longitude to x coordinate
+  final x = longitude * (conversionEarthRadius * pi / 180.0);
+  
+  // Convert latitude to y coordinate
+  final latRad = latitude * (pi / 180.0);
+  final y = log(tan((pi / 4) + (latRad / 2))) * conversionEarthRadius;
+  
+  // Clamp to valid Mercator bounds
+  final clampedX = max(min(x, mercatorLimit), -mercatorLimit);
+  final clampedY = max(min(y, mercatorLimit), -mercatorLimit);
+  
+  return [clampedX, clampedY];
```

Comment:
one important thing when converting to Position return type is that we support a third parameter after lng and lat which is altitude (alt).